esrally/driver/runner.py

```diff
@@ -593,7 +593,7 @@ def __call__(self, es, params):
     def request_body_query(self, es, params):
         request_params = params.get("request-params", {})
         if "cache" in params:
-            request_params["request_cache"] = params["cache"]
+            request_params["request_cache"] = str(params["cache"]).lower() if isinstance(params["cache"], bool) else params["cache"]
```

Comment:
simplify the right hand side to `str(params["cache"]).lower()`?

Reply:
Addressed in b0d6c4192f88ffaf32a8e5842d5d7568cba3ecfb